addons/sourcemod/scripting/gokz-core/teleports.sp

```diff
@@ -420,13 +476,27 @@ void TeleportToStart(int client)
 
 void TeleportToSearchStart(int client, int course)
 {
+	if (!CanTeleportToStart(client, true))
+	{
+		return;
+	}
+
 	// Call Pre Forward
 	Action result;
 	Call_GOKZ_OnTeleportToStart(client, course, result);
 	if (result != Plugin_Continue)
 	{
 		return;
 	}
+
+	// Safeguard Check
+	if (GOKZ_GetCoreOption(client, Option_Safeguard) > Safeguard_Disabled && GOKZ_GetTimerRunning(client) && GOKZ_GetValidTimer(client))
```

Comment:
Isn't this handeled by the check in `CanTeleportToStart` already?

Reply:
Right, I forgot to delete that. That should be fixed in the next commit.